connector/countconnector/factory.go

```diff
@@ -40,20 +47,7 @@ func NewFactory() connector.Factory {
 
 // createDefaultConfig creates the default configuration.
 func createDefaultConfig() component.Config {
-	return &Config{
-		Traces: MetricInfo{
-			Name:        defaultMetricNameSpans,
-			Description: defaultMetricDescSpans,
-		},
-		Metrics: MetricInfo{
-			Name:        defaultMetricNameDataPoints,
-			Description: defaultMetricDescDataPoints,
-		},
-		Logs: MetricInfo{
-			Name:        defaultMetricNameLogRecords,
-			Description: defaultMetricDescLogRecords,
-		},
-	}
+	return &Config{}
```

Comment:
Now using a custom unmarshaler to set defaults so that the default metric for each type can be disabled.